src/Tribe/Commerce/PayPal/Custom_Argument.php

```diff
@@ -57,6 +57,20 @@ public static function decode( $encoded, $assoc_array = false ) {
 			$encoded = str_replace( '\"', '"', $encoded );
 		}
 
+		// in case we receive the param in non json 'user_id:0,tribe_handler:tpp,pid:4' format
+		if ( false === strpos( $encoded, '{' ) && false === strpos( $encoded, '}' ) ) {
+			// we create the json from that string
+			$encoded = explode( ',', $encoded );
+
+			$encoded = array_reduce( $encoded, function ( $array, $item ) {
```

Comment:
We can't use anonymous functions on PHP 5.2. we need to setup a method and pass it as a `array()` callable here.